tsdb/index/index.go

```diff
@@ -802,17 +839,18 @@ func ReadSymbols(bs ByteSlice, version int, off int) ([]string, map[uint32]strin
 
 // ReadOffsetTable reads an offset table and at the given position calls f for each
 // found entry. If f returns an error it stops decoding and returns the received error.
-func ReadOffsetTable(bs ByteSlice, off uint64, f func([]string, uint64) error) error {
+func ReadOffsetTable(bs ByteSlice, off uint64, f func([]string, uint64, int) error) error {
 	d := encoding.NewDecbufAt(bs, int(off), castagnoliTable)
+	startLen := d.Len()
 	cnt := d.Be32()
 
 	// The Postings offset table takes only 2 keys per entry (name and value of label),
```

Comment:
This comment should be moved below, right before the `keys := make([]string, 0, 2)`

Reply:
Done.

I want to remove all the multi-index code as it's unused and presently confusing.